cpp/src/slice2java/Gen.cpp

```diff
@@ -91,43 +91,42 @@ namespace
     /// Returns a javadoc formatted link to the provided Slice identifier.
     string javaLinkFormatter(const string& rawLink, const ContainedPtr& source, const SyntaxTreeBasePtr& target)
     {
+        string sourceScope = getPackage(source);
+
         ostringstream result;
         result << "{@link ";
 
-        if (target)
```

Comment:
I also flattened out this `if` tree.
Unfortunately, the diff is struggling with it.